File: c/firstnseries.c
#include <stdio.h>

int main(void) 
{
  int Count, Term = 1, Counter, Multiple;
  printf("How many term in a series do you want to print?");
  scanf("%d", &Count);
  printf("The first %d terms in a series are", Count);
  for(Multiple = 1; Multiple <= Count; Multiple++)
  {
     Term = 1;
     for(Counter = 1; Counter <= Multiple; Counter++)
     {
       Term = Term * Multiple;
     }
     printf(" %d", Term);
  }
  printf(".\n");
  return 0;
}

File: python/firstnseries.py
Count = int(input("How many terms in a series do you want to print?"))
print('The first {0} terms in the series are:' .format(Count), end = "")
Multiple = 1
while(Multiple <= Count):
  Term = 1
  for Counter in range(1, Multiple + 1):
    Term = Term * Multiple
  print ("", Term, end = "")
  Multiple = Multiple + 1
print(".")